python/src/aiconfig/editor/client/src/components/prompt/prompt_input/schema_renderer/PromptInputAttachmentsSchemaRenderer.tsx

```diff
@@ -38,7 +40,7 @@ function EditableAttachmentRenderer({
           onRemoveAttachment={onRemoveAttachment}
           onEditAttachment={() => setShowUploader(true)}
         />
-      ) : (
+      ) : !readOnly && (
         <AttachmentUploader
```

Comment:
If they have a prompt that takes attachments but it has none, will this just be a blank prompt? Might be better to just render the AttachmentUploader still and have the dropzone disabled to denote that it's an input for attachments?

Reply:
I don't feel too strongly here -- can you compare how they look?

Reply:
<img width="975" alt="Screenshot 2024-01-19 at 10 19 56 AM" src="https://github.com/lastmile-ai/aiconfig/assets/141073967/e3cee944-0267-45bc-915c-723972935156">

disabled doesn't change how its rendered but only disables the dropzone. 
<img width="986" alt="Screenshot 2024-01-19 at 10 22 55 AM" src="https://github.com/lastmile-ai/aiconfig/assets/141073967/42d6e35a-e583-4ff2-bba6-da62b2ff68f1">

Reply:
I think the disabled dropzone is better than blank for now, otherwise there's no indication of what the input is 'supposed to be'. We do the same in lastmile so it is consistent as well

Reply:
updated